README_Deploy.md

```diff
@@ -0,0 +1,22 @@
+# Deploy of portal.qfnetwork.xyz
+
+Reminder please do no change this path `/root/cd/apps` on the server. This directory is (would be used) using for `ansible` deploy.
+
+## Manual deploy instruction
+
+The first please request access to the portal server via [request access instraction](https://github.com/QuantumFusion-network/infra/blob/main/docs/key_management_basic.md).
+
+```
+$> ssh 86.104.75.24
```

Comment:
Should we raplace that with <SERVER_IP> and not hardcode current IP address.

Maybe `frontend.qfnetwork.xyz` based on that infra config:
https://github.com/QuantumFusion-network/infra/blob/main/confs/ssh/config#L53